README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
+
+- Hit 'F5' to run your code. Click the link to upsert the output array values in your SQL table. Your upserted values should launch in the browser.
+- Congratulations! You have successfully created your first SQL output binding! Checkout [Output Binding Samples](#Output-Binding-Samples) for more information on how to use it and explore on your own!
+
+### Trigger Tutorial ###
+
+This tutorial requires that the Azure SQL database is setup as shown in [Create Azure SQL Database](#Create-Azure-SQL-Database), and that you have the 'Employee.cs' class from the [Input Binding Tutorial](#Input-Binding-Tutorial).
+
+- Create a new file
+- Add the following namespaces
+
+    ```csharp
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using System.Collections.Generic;
+    using Microsoft.Azure.WebJobs.Extensions.Sql;
+    ```
+
+- Below, add the SQL trigger
+
+    ```csharp
+    namespace Company.Function
+    {
+        public static class EmployeeTrigger
+        {
+            [FunctionName("EmployeeTrigger")]
+            public static void Run(
+                [SqlTrigger("[dbo].[Employees]", ConnectionStringSetting = "SqlConnectionString")]
+                IEnumerable<SqlChangeTrackingEntry<Employee>> changes,
+                ILogger logger)
+            {
+                foreach (var change in changes)
+                {
+                    Employee employee = change.Data;
+                    logger.LogInformation($"Change occurred to Employee table row: {change.ChangeType}");
+                    logger.LogInformation($"EmployeeID: {employee.EmployeeId}, FirstName: {employee.FirstName}, LastName: {employee.LastName}, Company: {employee.Company}, Team: {employee.Team}");
+                }
+            }
+        }
+    }
     ```
 
-1. See samples below for information on how to use the binding.
+- Add the below to 'local.settings.json' in "Values"
+
+    ```json
+    "AzureWebJobsDashboard": "UseDevelopmentStorage=true"
+    ```
+
+- (*Skip this step if you have not completed the output binding tutorial*) Open your output binding file and modify some of the values (e.g. Change Team from 'Functions' to 'Azure SQL'). This will update the row when the code is run.
+- Hit 'F5' to run your code. Click the link for the http trigger to the output binding. You should see the log in VS Code update and tell you which row changed and what the data in the row is now.
+- Update, insert, or delete additional rows in your SQL table using the SQL query editor while the function app is running and observe the log updates.
+- Congratulations! You have successfully created your first SQL trigger! Checkout [Trigger Samples](#Trigger-Samples) for more information on how to use the trigger and explore on your own!
+
+## Samples ##
```

Comment:
nit: I would name this "More Samples" (since we already have some earlier, in the tutorial)